lib/gitgitgadget.ts

```diff
@@ -271,35 +271,12 @@ export class GitGitGadget {
         // update work repo from base
         await this.updateNotesAndPullRef(pr.baseOwner, pr.number, previousTag);
 
-        // Remove template from description (if template exists)
-        let prBody: string;
-        try {
-            let prTemplate =
-                await git(["show",
-                           "upstream/master:.github/PULL_REQUEST_TEMPLATE.md"],
-                          { workDir: this.workDir });
-            // github uses \r\n so make sure it is set
-            prTemplate = prTemplate.replace(/\r?\n/g, "\r\n");
-            prBody = pr.body.replace(prTemplate, "");
-        } catch {
-            prBody = pr.body;
-        }
-
-        if (!prBody.length) {       // reject empty description
-            throw new Error("A pull request description must be provided");
-        }
-
-        // if known, add submitter to email chain
-        const ccSubmitter = userInfo.email ? `\nCc: ${
-            userInfo.name} <${userInfo.email}>` : "";
-        const description = `${pr.title}\n\n${prBody}${ccSubmitter}`;
-
         const series =
             await PatchSeries.getFromNotes(this.notes, pr.pullRequestURL,
-                                           description, pr.baseLabel,
+                                           pr.title, pr.body, pr.baseLabel,
                                            pr.baseCommit, pr.headLabel,
                                            pr.headCommit, options,
-                                           userInfo.name);
+                                           userInfo.name, userInfo.email);
```

Comment:
fyi, `userInfo.email` may be null.  Yes the interface needs to be updated to reflect that or return an empty string (some other code does not like null).  Another argument for passing interfaces.  The new gitgitgadget.test.ts test is passing undefined for the email.

Reply:
When I tried to pass `null` in test, I got this compile error:
`error TS2345: Argument of type 'null' is not assignable to parameter of type 'string'`

This is again where I lack Typescript understanding.

I have tested and everything seems to work as intended for cases where email is returned by GitGitGadget as `null`.

If there is something I _need_ to change, what is the simplest fix?

Reply:
Testing locally gives mixed results so I don't have a quick fix and nothing for you to change.  @dscho Any preference for `IGitHubUser.email` to support null or always return an empty string if github returns a null?